modules/sdpolicy-am-gcp-compute-types.adoc

```diff
@@ -8,7 +8,8 @@
 {product-title} offers the following worker node types and sizes on Google Cloud that are chosen to have a common CPU and memory capacity that are the same as other cloud instance types:
 [NOTE]
 ====
-`e2`, `a2`, and `a3` instance types are available for CCS only.
+`a2`, `a3`, `e2`, `c3` and `n4` instance types are available for CCS only.
+`c3` and `n4` instance types are available for OSD version 4.18 and later.
```

Comment:
```suggestion
`c3` and `n4` instance types are available for {osd} version 4.18 and later.
```

Reply:
Side note, we don't seem to have a variable for the OSD acronym. Is this something we should have?